src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/CreateReferralHelperTest.java

```diff
@@ -0,0 +1,155 @@
+    }
+
+    @Test
+    void populateSingleHearingDetails() {
+        caseData = CaseDataBuilder.builder()
+            .withHearing("1", "test", "Judy", "Venue", List.of("Telephone", "Video"),
+                "length num", "type", "Yes")
+            .withHearingSession(0, "1", "2021-12-25T00:00:00.000",
+                HEARING_STATUS_POSTPONED, false)
+            .build();
+
+        createReferralHelper.populateHearingDetails(caseData);
+
+        assertThat(caseData.getReferralHearingDetails())
+            .isEqualTo(expectedSingleHearingDetails);
+    }
+
+    @Test
+    void populateMultipleHearingDetails() {
+        caseData = CaseDataBuilder.builder()
+            .withHearing("1", "test", "Judy", "Venue", List.of("Telephone", "Video"),
+                "length num", "type", "Yes")
+            .withHearingSession(0, "1", "2021-12-25T00:00:00.000",
+                HEARING_STATUS_POSTPONED, false)
+            .withHearing("1", "test", "Judy", "Venue", List.of("Telephone", "Video"),
+                "length num", "type", "Yes")
+            .withHearingSession(1, "1", "2021-12-26T00:00:00.000",
+                HEARING_STATUS_HEARD, false)
+            .build();
+
+        createReferralHelper.populateHearingDetails(caseData);
+
+        assertThat(caseData.getReferralHearingDetails())
+            .isEqualTo(expectedMultipleHearingDetails);
+    }
+
+    @Test
+    void saveReferralToTheReferralCollection() {
+        DocumentTypeItem document1 = new DocumentTypeItem();
+        document1.setId("1");
+        DocumentTypeItem document2 = new DocumentTypeItem();
+        document2.setId("2");
+        caseData.setReferCaseTo("Judge Judy");
+        caseData.setIsUrgent("Yes");
+        caseData.setReferentEmail("judge.judy@aol.com");
+        caseData.setReferralSubject("Subject line here");
+        caseData.setReferralSubjectSpecify("Custom subject line");
+        caseData.setReferralDetails("This is an explanation");
+        caseData.setReferralDocument(List.of(document1, document2));
+        caseData.setReferralInstruction("Custom instructions for judge");
+
+        createReferralHelper.createReferral(caseData, "");
+
+        String expected = "ReferralType(referralNumber=1, referralHearingDate=None, referCaseTo=Judge Judy, "
+            + "referentEmail=judge.judy@aol.com, isUrgent=Yes, "
+            + "referralSubject=Subject line here, referralSubjectSpecify=Custom subject line, referralDetails=This "
+            + "is an explanation, referralDocument=[DocumentTypeItem(id=1, value=null), DocumentTypeItem(id=2, "
+            + "value=null)], referralInstruction=Custom instructions for judge, referredBy=Judge Judy, "
+            + "referralDate="
+            + Helper.getCurrentDate()
+            + ", referralStatus=Awaiting instructions)";
+        String actual = caseData.getReferralCollection().get(0).getValue().toString();
+        assertEquals(expected, actual);

```

Comment:
Minor - mix of assertThat().isEqualTo() and assertEquals, maybe just stick with one assertion?